crates/gpui/src/platform/blade/blade_renderer.rs

```diff
@@ -178,7 +180,10 @@ impl BladePipelines {
                 depth_stencil: None,
                 fragment: Some(shader.at("fs_quad")),
                 color_targets,
-                multisample_state: gpu::MultisampleState::default(),
+                multisample_state: gpu::MultisampleState {
+                    sample_count,
```

Comment:
Not sure if we need to enable the MSAA for pipelines other than the one for Paths. As far as I understand all the other primitives don't need it, since they already perform their own AA on per-pixel level in fragment shaders. They might be incompatible with MSAA anyway, because we are not using alpha-to-coverage. But I could be wrong, I don't know the inner workings of GPUI well. :slightly_smiling_face: